### ISSUE TITLE
[Toggle button] Disabled button is not announced as "disabled"

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3102]
## Current behavior 

Screen reader user placing their focus on [disabled toggle button](https://storybook.sanomalearning.design/?path=/story/actions-toggle-button--disabled) gets announcement "Show settings, toggle button". The part about button being disabled is missing.

It's caused by using `disabled` attribute on `sl-toggle-button` is not passed to Accessibility Tree and screen reader is not getting this information announced.

<img width="257" height="283" alt="Image" src="https://github.com/user-attachments/assets/80f1b7c7-6057-4575-b6e7-83108a872630" />

Problematic disabled toggle buttons occurs also in [All story](https://storybook.sanomalearning.design/?path=/story/actions-toggle-button--all), and should be changed there as well.

## Suggested solution

It's recommended to change `disabled` to `aria-disabled` on `sl-toggle-button` as it would result in toggle button being announced as "Show settings, toggle button, dimmed".
Right now disabled `sl-toggle-button` has `tabindex=0` and is focusable with keyboard. With `aria-disabled` toggle button can still have this behavior, because this would be consistent with other components from DS that use the same behvaior.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
